TGM/src/main/java/network/warzone/tgm/map/MapInfoDeserializer.java

```diff
@@ -50,7 +50,9 @@ public MapInfo deserialize(JsonElement jsonElement, Type type, JsonDeserializati
             ChatColor teamColor = ChatColor.valueOf(teamJson.get("color").getAsString().toUpperCase().replace(" ", "_"));
             int teamMax = teamJson.get("max").getAsInt();
             int teamMin = teamJson.get("min").getAsInt();
-            parsedTeams.add(new ParsedTeam(teamId, teamName, teamColor, teamMax, teamMin));
+            int respawnDelay = 3;
+            if (teamJson.has("respawnDelay")) respawnDelay = teamJson.get("respawnDelay").getAsInt();
+            parsedTeams.add(new ParsedTeam(teamId, teamName, teamColor, teamMax, teamMin, respawnDelay));
```

Comment:
Setting the respawn delay in the team JSON object is probably not the best approach. Also, the respawn delay should be parsed using milliseconds since even a tenth of a second can make a difference in a fast-paced game

Perhaps a system with support for multiple respawn rules? Example:
```JSON

    "respawn": {
        "rules": [
            {
                "teams": ["red"],
                "delay": 2500,
                "freeze": true,
                "blindness": false,
                "confirm": true
            },
            {
                "teams": ["blue"],
                "delay": 2000,
                "freeze": false,
                "blindness": true,
                "confirm": true
            }
        ]
    }

```